src/components/Reactions/EmojiReactionBubble.js

```diff
@@ -38,22 +38,36 @@ const propTypes = {
      */
     hasUserReacted: PropTypes.bool,
 
+    /** Whether the user can react with the current emoji */
```

Comment:
Not sure this comment is doing much.. maybe we can provide something that explains why we block emoji reactions?

```
    /** We disable reacting with emojis on comments that have... (etc.) */
```